src/mal/mal/mal_module.c

```diff
@@ -28,6 +28,7 @@
  */
```

Comment:
where do these changes come from? default?

Reply:
Mine in MonetDBJavaLite. I was having a race condition there while shutting down the database in one of the stress tests.